aten/src/ATen/functorch/PlumbingHelper.h

```diff
@@ -26,6 +26,8 @@
 
 namespace at { namespace functorch {
 
+void vmap_check_escaped(optional<DynamicLayer> layer, const char* what);
```

Comment:
Should pass by reference instead of pass by value (pass by value copies the object, which is not what we want). i.e., `const optional<DynamicLayer>& layer, ...`